templates/cisco_nxos_show_interface_transceiver.textfsm

```diff
@@ -0,0 +1,20 @@
+Value INTERFACE (\S+/\S+)
+Value MANUFACTURER (\S+)
+Value TYPE (.+)
+Value SERIAL (\S+)
+Value PART_NUMBER (\S+)
+
+Start
+  ^${INTERFACE}
+  ^\s+type\s+is\s+${TYPE}
+  ^\s+name\s+is\s+${MANUFACTURER} 
+  ^\s+part\s+number\s+is\s+${PART_NUMBER}
+  ^\s+serial\s+number\s+is\s+${SERIAL} -> Record
+  ^\s+transceiver\s+
+  ^\s+nominal\s+
+  ^\s+revision\s+
+  ^\s+Link\s+
+  ^\s+cisco\s+id
+  ^\s+cisco\s+extended
+  ^\s*$$
+  ^. -> Error
```

Comment:
```suggestion
  ^. -> Error
  
```

Reply:
Thanks @jmcgill298 I made the changes as suggested